### PR TITLE
Add TIL for memory management flags in node

### PR DESCRIPTION
**Pull request by saminakh**

### CODE REVIEW

File: node/low-mem-env.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+If you need to reproduce a memory error locally, where your environment is very different from staging or production servers, it turns out that node has some useful tools in doing so! For example, my local setup does not have the database size to recreate a memory issue encountered on a production server, but I can simulate a low memory environment using a flag to limit the amount of memory available to node: 
+
+```
+node --max_old_space_size=128 index.js
+```
+
+This will limit us to 128 Mb when running `index.js` in my npm start script. You can increase or even decrease this value further, or configure additional limitations using some of the other flags available for node: 
+
+```
+http://erikcorry.blogspot.ru/2012/11/memory-management-flags-in-v8.html
```
Comment (wdrach) [inline node/low-mem-env.md]: shouldn't this be a link, not a code block? e.g.

`[other flags here](http://erikcorry.blogspot.ru/2012/11/memory-management-flags-in-v8.html)`

Which results in:
[other flags here](http://erikcorry.blogspot.ru/2012/11/memory-management-flags-in-v8.html)

Comment (saminakh) [inline node/low-mem-env.md]: I have no idea what you're talking about!
...
...
😛

**Comment by wdrach** [comment]
:shipit:

**Comment by staylorwr** [comment]
@saminakh you know you can create your own branches right?  :clap: to all

**Comment by saminakh** [comment]
@staylorwr i'm confused...i did create my own branch?

**Comment by staylorwr** [comment]
yeah, did you do some strange rebasing or something?  There's a @tdrach commit in here.

**Comment by saminakh** [comment]
i did a fixup b/c of will's suggestion, don't really know what happened. https://github.com/MotelIs/til/blame/master/css/pseudo-element-unicode.md looks fine though so 😕

**Pull request merged**